tests/Nox.Types.Tests/Types/Image/ImageTests.cs

```diff
@@ -0,0 +1,165 @@
+using FluentValidation;
```

Comment:
Can we write the tests using FluentAssertions, as it is specified herehttps://regusit.atlassian.net/wiki/spaces/NOX/pages/3153264693/Test+Projects?

Reply:
Required changes were applied